src/cache/cache_las.c

```diff
@@ -796,6 +796,9 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
             WT_ERR(__las_insert_record(session, cursor, btree_id, key, upd, &stop_ts, &stop_txnid));
             ++insert_cnt;
 
+            /* Flag the update as now in the lookaside file. */
+            F_SET(upd, WT_UPDATE_HISTORY_STORE);
```

Comment:
I think we can't set the flag here as the transaction hasn't been committed. If we roll back, we will lose the updates.

Reply:
And I think we only need to mark the newest version in lookaside with that flag.